### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by stefanateljevic04** [merrickliu888/NutriNinja#38]
After we have finished the implementation and our minimum viable product has been reached, we should consider some accessibility options for users that may have trouble using our application. For example, we could add: 

- Dark mode / High contrast for users with colour blindness or other visual impairments
- Text resizing to give users the option to improve personal readability
- Text-to-speech / speech recognition for users with visual impairments
- Keyboard navigation for users that cannot use a mouse
